api/src/BcGov.Malt.Web/Controllers/UsersController.cs

```diff
@@ -74,9 +77,94 @@ public async Task<ActionResult<DetailedUser>> LookupAsync(string username)
 
             if (string.IsNullOrEmpty(username)) return BadRequest();
 
-            await Task.Delay(0); // to suppress warning CS1998: This async method lacks 'await'
+            var user = await _userSearchService.SearchAsync(username);
+
+            if (user != null)

```

Comment:
personal style nit: check the null case first to avoid amount of code needed inside conditional

```
if (user == null) return notfound;
// everything else
```

Reply:
The method parameter is username which is being checked, and then the username is being fetched from the service.  That is what is being done.